.github/workflows/deploy_prod.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy production
+
+on:
+  release:
+    types: [ released ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    env:
+      DOCKERHUB_ORG: gnosispm
+      DOCKERHUB_PROJECT: safe-client-gateway
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # checks out all branches and tags, necessary for versioning /about endpoint
```

Comment:
How does this affect the docker build that happens on `bash scripts/deploy_docker.sh`

Reply:
I think the reason I did this, was so that the script you mention would find the tag. I remember that with a regular checkout of the branch, tags were somehow not included. There is probably a nicer way to do this, as it sounds like a pretty common use case.